halo2/src/prover.rs

```diff
@@ -307,7 +544,7 @@ fn gen_proof<
         .map(|instances| instances.as_slice())
         .collect_vec();
     let proof = {
-        let mut transcript = TW::init(Vec::new());
+        let mut transcript = TranscriptWriterBuffer::<_, G1Affine, _>::init(Vec::new());
```

Comment:
```suggestion
        let mut transcript = TW::init(Vec::new());
```

I don't think this change is necessary?

Reply:
Will check